tests/bud.bats

```diff
@@ -242,24 +242,38 @@ expect_output --substring "hello"
   # With explicit and full --platform, buildah should not warn.
   run_buildah build $WITH_POLICY_JSON --platform linux/amd64/v2 \
               -t source -f $containerfile
-  assert "$output" !~ "missing .* build argument" \
-         "With explicit --platform, buildah should not warn"
+  assert "$output" =~ "image platform .* does not match the expected platform" \
+         "With explicit --platform, buildah should warn about no variant"
```

Comment:
This error description doesn't seem to match the error message it's checking against.